currency/manager.go

```diff
@@ -539,39 +538,30 @@ func (fs FullStore) MarshalJSON() ([]byte, error) {
 	return json.Marshal(temp)
 }
 
-// copy copies and segregates pair store from internal and external calls.
-func (ps *PairStore) copy() (*PairStore, error) {
+// clone returns a deep clone of the PairStore
+func (ps *PairStore) clone() *PairStore {
 	if ps == nil {
-		return nil, errPairStoreIsNil
+		return nil
```

Comment:
The removal of the error here and making returning nil an acceptable situation is something that could lead to panics as not much code is prepared for that. 

At present it looks like only a botched test via `Load` will lead to this which is acceptable and addressable by the offending user, but I still need to point out because 🦕

Reply:
Understood.